backend/api/Services/MissionRunService.cs

```diff
@@ -198,7 +198,7 @@ private static void SearchByName(ref IQueryable<MissionRun> missionRuns, string?
 
             missionRuns = missionRuns.Where(
                 missionRun =>
-                    missionRun.Name != null && missionRun.Name.Contains(name.Trim(), StringComparison.OrdinalIgnoreCase)
+                    missionRun.Name != null && missionRun.Name.Contains(name.Trim())
```

Comment:
```suggestion
                    missionRun.Name != null && missionRun.Name.ToLower().Contains(name.ToLower().Trim())
```

Reply:
Suggestion: Make sure that the robots and tags appear regardless of letter case.